controller/workitemtype_blackbox_test.go

```diff
@@ -154,24 +153,17 @@ func (s *workItemTypeSuite) createWorkItemTypeAnimal() (http.ResponseWriter, *ap
 	s.T().Log("Creating 'animal' work item type...")
 	responseWriter, wi := test.CreateWorkitemtypeCreated(s.T(), nil, nil, s.typeCtrl, space.SystemSpace, &payload)
 	require.NotNil(s.T(), wi)
-	require.NotNil(s.T(), wi.Data)
-	require.NotNil(s.T(), wi.Data.ID)
-	require.True(s.T(), uuid.Equal(animalID, *wi.Data.ID))
```

Comment:
We do check for the whole "animal" WIT [here](https://github.com/almighty/almighty-core/pull/1419/files/a74b0d396840844b70017055618980520f721b60#diff-b86e87864d1430ad31dcf40321747fa7R267) and compare it against the golden file.